mods/content/polaris/datum/languages/human.dm

```diff
@@ -112,11 +112,111 @@
 		"wa", "wa", "wa", "wa", "wa", "wa", "wa", "wai", "wan", "wang", "wei", "wen", "weng", "wo", "wu",
 		"xi", "xia", "xian", "xiang", "xiao", "xie", "xin", "xing", "xiong", "xiu", "xu", "xuan", "xue", "xun",
 		"ya", "yan", "yang", "yao", "ye", "yeg", "yeg", "yeg", "yey", "yey", "yey", "yi", "yin", "ying", "yong", "you", "yu", "yuan", "yue", "yun",
-		"za", "zai", "zan", "zang", "zao", "ze", "zei", "zen", "zeng", "zi", "zong", "zou", "zu", "zuan", "zui", "zun", "zuo", 
+		"za", "zai", "zan", "zang", "zao", "ze", "zei", "zen", "zeng", "zi", "zong", "zou", "zu", "zuan", "zui", "zun", "zuo",
 		"zha", "zhai", "zhan", "zhang", "zhao", "zhe", "zhei", "zhen", "zheng", "zhi",
 		"zhong", "zhou", "zhu", "zhua", "zhuai", "zhuan", "zhuang", "zhui", "zhun", "zhuo"
 	)
 
+/decl/language/human/tradeband
+	name = "Tradeband"
+	uid = "language_tradeband"
+	desc = "Spoken by the humans of the upper-class Sagittarius Heights, Tradeband was designed to be pleasing to both humans and their Skrellian trading partners."
+	speech_verb = "enunciates"
+	colour = ""
+	key = "2"
+	partial_understanding = list("Galactic Common" = 40, "Solar Common" = 20, "Terminus" = 40, "Gutter" = 10, "Sivian Creole" = 10)
+	//, LANGUAGE_SKRELLIAN = 40, LANGUAGE_SKRELLIANFAR = 15
+	syllables = list(
+		"fea","vea","vei","veh","vee","feh","fa","soa","su","sua","sou","se","seh","twa","twe","twi",
+		"ahm","lea","lee","nae","nah","pa","pau","fae","fai","soh","mou","ahe","ll","ea","ai","thi",
+		"hie","zei","zie","ize","ehy","uy","oya","dor","di","ja","ej","er","um","in","qu","is","re",
+		"nt","ti","us","it","en","at","tu","te","ri","es","et","ra","ta","an","ni","li","on","or","se",
+		"am","ae","ia","di","ue","em","ar","ui","st","si","de","ci","iu","ne","pe","co","os","ur","ru"
+	)
+
+
+// Criminal language.
+/decl/language/human/gutter
+	name = "Gutter"
+	uid = "language_gutter"
+	desc = "Gutter originated as a Thieves' Cant of sorts during the early colonization era. The language eventually spread from the cartels and triads to the disenfranchised people of the Bowl."
+	speech_verb = "growls"
+	colour = ""
+	key = "3"
+	space_chance = 45
+	partial_understanding = list("Galactic Common" = 20, "Solar Common" = 20, "Tradeband" = 10, "Terminus" = 20, "Sivian Creole" = 15)
+	//LANGUAGE_SKRELLIAN = 15, LANGUAGE_SIIK = 10) also
```

Comment:
Same here, Catlanguage should already exist but is it inaccessible due to being in another modpack currently?